war/src/main/js/filter-build-history.js

```diff
@@ -159,8 +159,8 @@ function updatePageParams(dataTable) {
   );
 }
 function togglePageUpDown() {
-  buildHistoryPageNav.classList.remove("hasUpPage");
-  buildHistoryPageNav.classList.remove("hasDownPage");
+  buildHistoryPageNav?.classList.remove("hasUpPage");
+  buildHistoryPageNav?.classList.remove("hasDownPage");
   if (hasPageUp()) {
     buildHistoryPageNav.classList.add("hasUpPage");
   }
```

Comment:
I would suggest to have a variable indicating "noPageNav" or equivalent instead of using `?.`. It could be confusing, especially in such method with a mix of `?.` and `.` that's reducing readability of the code (at first glance it seems to be a mistake)